src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/TdsParserSafeHandles.cs

```diff
@@ -156,11 +180,21 @@ internal sealed class SNIHandle : SafeHandle
                 instanceName = new byte[256]; // Size as specified by netlibs.
                 if (ignoreSniOpenTimeout)
                 {
+                    // UNDONE: ITEM12001110 (DB Mirroring Reconnect) Old behavior of not truly honoring timeout presevered 
+                    //  for non-failover scenarios to avoid breaking changes as part of a QFE.  Consider fixing timeout
+                    //  handling in next full release and removing ignoreSniOpenTimeout parameter.
                     timeout = Timeout.Infinite; // -1 == native SNIOPEN_TIMEOUT_VALUE / INFINITE
                 }
 
+#if NETFRAMEWORK
+                int transparentNetworkResolutionStateNo = (int)transparentNetworkResolutionState;
```

Comment:
I don't know what the plans are for transparent network resolution. It may be a feature that should come to netcore eventually. It might be an idea to have a new compiler constant for it, @David-Engel is probably the person who would know.

Reply:
The current sentiment around TNIR is that we can bring connection string and SqlConnectionStringBuilder parity to netcore but we should not actually implement the TNIR logic. Apparently it caused more support issues than it solved for Azure. Customers that really need the driver to initiate parallel connection attempts against each IP DNS resolves to should use MSF.